### ISSUE TITLE
Balance Plague Bringer / Inflict Poison

### ISSUE THREAD

**Issue by jdoleary** [jdoleary/Spellmasons#1076]
mxMeeple
OP
 — 09/04/2024 8:18 PM
TLDR: Plague Bringer is too strong, Inflict Poison is too weak
Plague Bringer rune is 40 per, to apply 20 poison to X units. Which in itself is insane cost ratio (combined with Heavy Toxins is just auto-play).
Inflict Poison rune is 80 per, to apply 1 poison to damaged units. You have to craft an insane spell just to compete with Plague Bringer, not to mention dumping hundreds to even make it worth while. 
I was so excited to try Bolt with Inflict Poison but its completely useless even with choice runes. 
My current run is Plague Bringer 10, Heavy Toxins 3 and I can almost just AFK

---
Xulqelyat — 09/04/2024 8:37 PM
I thought it would be funny to request plauge bringer to work when you are dead like a diseased corpse 
@Creator @SoulMuncher  can we make plauge bringer be in 5 dmg increments and cost like 200sp maybe and only infect stuff in the player cast range.

and for inflict poison should also be increased to like 5dmg and maybe see about adding some % of damage x curse effect runes
mxMeeple
OP
 — 09/04/2024 8:40 PM
Maybe only 2 of those nerfs for Plague Bringer
